validation/optimism-portal-2-params_test.go

```diff
@@ -0,0 +1,46 @@
+package validation
+
+import (
+	"context"
+	"testing"
+
+	bindings "github.com/ethereum-optimism/optimism/op-node/bindings/preview"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation/standard"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testOptimismPortal2Params(t *testing.T, chain *ChainConfig) {
+	skipIfExcluded(t, chain.ChainID)
+	opAddr, err := Addresses[chain.ChainID].AddressFor("OptimismPortalProxy")
+	require.NoError(t, err)
+
+	rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+
+	require.NotEmpty(t, rpcEndpoint, "no public endpoint for chain")
+	client, err := ethclient.Dial(rpcEndpoint)
+	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+	op, err := bindings.NewOptimismPortal2(common.Address(opAddr), client)
+	require.NoError(t, err)
+
+	callOpts := &bind.CallOpts{Context: context.Background()}
```

Comment:
Do we always use the background ctx in those checks? Wouldn't it actually make sense to use some sane network timeout so that those tests fail quickly on a broken RPC connection?

Reply:
Yeah I think that's a good suggestion, I did that here [42ecdc4](https://github.com/ethereum-optimism/superchain-registry/pull/383/commits/42ecdc416c3c45e7a03f9e843427e8fad7ae9722). I will circle back in a future PR and pull these options out to a shared location so that all validation checks use the same timeout.